transport-native-epoll/build.gradle

```diff
@@ -0,0 +1,63 @@
+plugins {
+  id 'com.android.library'
+  id 'kotlin-android'
+  id 'maven-publish'
+}
+
+android {
+  compileSdkVersion 29
+  buildToolsVersion "30.0.2"
+
+  defaultConfig {
+    minSdkVersion 21
+    targetSdkVersion 29
+    versionCode 1
+    versionName "1.0"
+    testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+  }
+  compileOptions {
+    sourceCompatibility JavaVersion.VERSION_1_8
+    targetCompatibility JavaVersion.VERSION_1_8
+  }
+  kotlinOptions {
+    jvmTarget = '1.8'
+  }
+  externalNativeBuild {
+    cmake {
+      path 'CMakeLists.txt'
+    }
+  }
+}
+
+dependencies {
+  api 'io.netty:netty-transport-native-epoll:4.1.55.Final'
+}
+
+// `./gradlew publishToMavenLocal` to publish
```

Comment:
this is required to be done manually before usage. Also it requires adding `mavemLocal()` to consumer project. As [a probably better alternative] one can add CI build config to build it with Gradle (after Maven) or update Maven build configuration